book/10-git-internals/sections/maintenance.asc

```diff
@@ -1,29 +1,30 @@
-=== Maintenance and Data Recovery
+=== Pagpapanatili At Pagbalik ng Datos
 
-Occasionally, you may have to do some cleanup – make a repository more compact, clean up an imported repository, or recover lost work.
-This section will cover some of these scenarios.
+Paminsan-minsan ay maaring kailangan mong magsagawa ng paglinis - gawing mas masinsin ang repositoryo, paglinis ng inangkat na repositoryo, o pagbalik ng nawalang trabaho. Ang ilang mga sitwasyon ay sakop ng seksyong ito.
 
 [[_git_gc]]
-==== Maintenance
+==== Pagpapanatili
 
-Occasionally, Git automatically runs a command called ``auto gc''.
-Most of the time, this command does nothing.
-However, if there are too many loose objects (objects not in a packfile) or too many packfiles, Git launches a full-fledged `git gc` command.
-The ``gc'' stands for garbage collect, and the command does a number of things: it gathers up all the loose objects and places them in packfiles, it consolidates packfiles into one big packfile, and it removes objects that aren't reachable from any commit and are a few months old.
+Paminsan-minsan, awtomatikong tinatawag ng Git ang utos na ``auto gc''.
+Sa halos lahat ng panahon, ang utos na ito ay walang ginagawa.
+Subalit, kung mayroong maraming loose na mga object (mga object na hindi packfile) o napakaraming mga packfile, ang Git ay naglulunsad ng isang buong `git gc` na utos.
+Ang ``gc'' ay kumakatawan sa pagkolekta ng basura, at ang utos na ito ay nagsasagawa ng maraming bagay: tinitipon nito ang lahat ng mga loose na mga object at inilalagay sa mga packfile, pinagsasama nito ang mga packfile sa isang malaking packfile, at inaalis nito ang mga bagay na hindi naabot nang anumang commit at ilang buwan na ang edad.
 
-You can run auto gc manually as follows:
+Maaari mong patakbuhin ang auto gc nang mano-mano tulad ng sumusunod:
 
 [source,console]
 ----
 $ git gc --auto
 ----
 
-Again, this generally does nothing.
-You must have around 7,000 loose objects or more than 50 packfiles for Git to fire up a real gc command.
-You can modify these limits with the `gc.auto` and `gc.autopacklimit` config settings, respectively.
+Sa uulitin, ito ay walang ginagawa.
+Kailangan ay mayroon kang humigit kumulang na 7,000 na loose na mga object or humigit sa 50 na packfile para  
+You must have around 7,000 loose objects or more than 50 packfiles para isagawa ng Git ang isang tunay na gc na utos.
```

Comment:
Please remove untranslated line.
`You must have around 7,000 loose objects or more than 50 packfiles`

Reply:
Corrected.  :)